peachjam/js/components/FilterFacets/SingleFacet.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <template v-if="facet.options && facet.options.length">
+    <li class="list-group-item">
+      <div class="d-flex justify-content-between mb-2">
+        <strong>{{ facet.title }}</strong>
+        <div class="d-flex align-items-center">
+          <a
+            v-if="facet.type === 'checkboxes' && facet.value.length"
+            href="#"
+            @click.prevent="$emit('clear-facet', facet.name)"
+          >
+            {{ $t('Clear') }}
+          </a>
+
+          <a
+            v-if="facet.value && facet.type !== 'checkboxes'"
+            href="#"
+            @click.prevent="$emit('clear-facet', facet.name)"
+          >
+            {{ $t('Clear') }}
+          </a>
```

Comment:
Lots of duplication here -- perhaps make the condition a computed property like `clearable` or `hasValue` which does the complex boolean logic so there's only one `a` tag.

Reply:
Just like `showClearAllFilter` in FilterFacets/index.vue